server/server.go

```diff
@@ -8,11 +8,11 @@ import (
 	"github.com/milehighgophers/website/ui"
 )
 
-type Store interface {
-	AllEvents() *data.MeetupSchedule
-}
+// type Store interface {
```

Comment:
I wouldn't commit code that is commented out. If it isn't needed just delete it. It is in version control so if it is needed in the future it can be brought back.

Reply:
I think I just missed this when I committed -- I can rebase this out.